GitUI/UserControls/RevisionFilter.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace GitUI.UserControls
+{
+    public class RevisionFilter
+    {
+        // TODO: consider using a bit vector or an enum
```

Comment:
or make it a `record`

Reply:
Yeah-nah :) I'm not planning to do it now.... There are other low hanging fruits we can pluck before this one.